x/bank/internal/handler/handler.go

```diff
@@ -1,15 +1,16 @@
-package bank
+package handler
```

Comment:
hmm shouldn't this be under `internal` ?

Reply:
Nope, it's needs to be handler. I could move it under types

Reply:
> Nope, it's needs to be handler. 

is it because a restriction from `internal` that it can't live under`x/bank/internal/handler.go` ? I'm fine with it as long as we update #4438

Reply:
I'm unsure about using `internal` just like other normal packages, i.e. dropping `.go` files in `internal` may not be permitted as internal may not compile.

Reply:
Hmmm, don't we want files like `abci.go` and `handler.go` to live in the parent/root of the module and _not_ under `internal`? I.e. I wouldn't like to see a module turn into everything being shoved into `internal` with just an `alias.go` at the root.